unit_tests/stencil_composition/sid/test_sid_helpers.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <gridtools/stencil_composition/sid/sid_helpers.hpp>
+
+#include <gtest/gtest.h>
+
+#include <gridtools/common/defs.hpp>
+#include <gridtools/common/hymap.hpp>
+#include <gridtools/stencil_composition/dim.hpp>
+#include <gridtools/stencil_composition/sid/synthetic.hpp>
+
+namespace gridtools {
+    namespace {
+        using sid::property;
+        using namespace literals;
+
+        TEST(sid_helpers, shifted) {
+            double data[3][5][7];
+
+            using strides_t = hymap::keys<dim::i, dim::j, dim::k>::
```

Comment:
No need to build `src` --  `data` is a SID already.

Reply:
That would mean testing with default keys? Actually this is copy-paste from the `test_sid_delegate.cpp`. Should I change it there as well?

Reply:
Yes. Why not. Also I think that using `dim` namespace here is a bad idea. It just create unwanted dependencies from outer `stencil_composition` stuff to the `SID` API